src/scripts/main.ts

```diff
@@ -113,6 +125,13 @@ function main(): void {
         '*://*.duckduckgo.com/*',
         '*://*.qwant.com/*',
         '*://*.startpage.com/*',
+        '*://*.ecosia.org/*',
+        '*://*.brave.com/*',
+        '*://*.swisscows.com/*',
```

Comment:
swisscows doesn't seem to work, please investigate or remove this domain from the PR